spec/betterc.dd

```diff
@@ -0,0 +1,48 @@
+Ddoc
+
+$(SPEC_S Better C,
+
+`-betterC` is a Command-Line flag for `dmd`, which generated light-weight,
+barebone, runtime-free binaries with are suitable for linking from C.
```

Comment:
Yes, Unwind_Resume and __assert are both emitted by the compiler almost always. It assumes a C lib, but it is fairly easy to pull that out too for special uses. Read my post here: http://forum.dlang.org/post/cwzmbpttbaqqzdetwkkf@forum.dlang.org